trame_server/core.py

```diff
@@ -421,6 +425,10 @@ def start(
         CoreServer.bind_server(self)
         options = self.cli.parse_known_args()[0]
```

Comment:
@jourdain, would `host` ever get set by `parse_known_args()`?

Reply:
yes it can be part of it...